src/core/client/mock_routing/mod.rs

```diff
@@ -682,136 +653,163 @@ impl Drop for MockRouting {
     }
 }
 
+fn update_account_info(storage: &mut Storage, client_name: &XorName) {
+    let account = storage.get_or_create_account_info(client_name);
+    account.data_stored += 1;
+    account.space_available -= 1;
+}
+
 #[cfg(test)]
 mod tests {
     use core::client::account::Account;
```

Comment:
We should not be using this in mock-routing - should be completely independent of other modules. We will soon be changing `Account` so will need to come back here to update. Better to avoid it altogether.